src/common/error/errors/missing-data-error.ts

```diff
@@ -57,6 +58,9 @@ function getErrorMessage(type: MissingDataErrorType): string {
     case MissingDataErrorType.MissingShippingAddress:
         return 'Unable to proceed because shipping address data is unavailable.';
 
+    case MissingDataErrorType.MissingSpamProtection:
+            return 'Unable to proceed because spam protection is not completed.';
```

Comment:
indentation off. Maybe this should be `Unable to proceed because spam protection token is not provided`

Since this is for developers, it could be that the challenge was rendered, but the developer is not passing it in the code itself.